From f9abac545cb3544f553c25b357fe2020d2f2c0ac Mon Sep 17 00:00:00 2001
From: Mehdi Sabwat <mehdisabwat@gmail.com>
Date: Thu, 5 Sep 2019 18:01:23 +0200
Subject: [PATCH 04/15] contrib: disable unsupported -fPIC in wasm backend

Enabling -fPIC will crash the compiler, so we need to disable it for now.
---
 contrib/src/main.mak | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/contrib/src/main.mak b/contrib/src/main.mak
index 8b65ffdd7b..18f4b038a9 100644
--- a/contrib/src/main.mak
+++ b/contrib/src/main.mak
@@ -310,9 +310,14 @@ ifdef HAVE_WIN32
 HOSTCONF += --without-pic
 PIC :=
 else
+ifdef HAVE_EMSCRIPTEN
+HOSTCONF += --without-pic
+PIC :=
+else
 HOSTCONF += --with-pic
 PIC := -fPIC
 endif
+endif
 
 # For cross-compilation with meson, do not set compiler and flags
 # in HOSTVARS as meson will always use them for the BUILD machine compiler!
-- 
2.23.0

